clients/unshaded/src/main/java/tachyon/client/file/FileSystem.java

```diff
@@ -0,0 +1,375 @@
+  boolean exists(TachyonURI path, ExistsOptions options)
+      throws InvalidPathException, IOException, TachyonException;
+
+  /**
+   * Convenience method for {@link #free(TachyonURI, FreeOptions)} with default options.
+   *
+   * @param path the path to free in Tachyon space
+   * @throws IOException if a non-Tachyon exception occurs
+   * @throws FileDoesNotExistException if the given path does not exist
+   * @throws TachyonException if an unexpected Tachyon exception is thrown
+   */
+  void free(TachyonURI path) throws FileDoesNotExistException, IOException, TachyonException;
+
+  /**
+   * Evicts any data under the given path from Tachyon space, but does not delete the data from the
+   * UFS.
+   *
+   * @param path the path to free in Tachyon space
+   * @param options options to associate with this operation
+   * @throws IOException if a non-Tachyon exception occurs
+   * @throws FileDoesNotExistException if the given path does not exist
+   * @throws TachyonException if an unexpected Tachyon exception is thrown
+   */
+  void free(TachyonURI path, FreeOptions options)
+      throws FileDoesNotExistException, IOException, TachyonException;
+
+  /**
+   * Convenience method for {@link #getStatus(TachyonURI, GetStatusOptions)} with default options.
+   *
+   * @param path the path to obtain information about
+   * @return the {@link URIStatus} of the file
+   * @throws IOException if a non-Tachyon exception occurs
+   * @throws FileDoesNotExistException if the path does not exist
+   */
+  URIStatus getStatus(TachyonURI path)
+      throws FileDoesNotExistException, IOException, TachyonException;
+
+  /**
+   * Gets the {@link URIStatus} object that represents the metadata of a Tachyon path.
+   *
+   * @param path the path to obtain information about
+   * @param options options to associate with this operation
+   * @return the {@link URIStatus} of the file
+   * @throws IOException if a non-Tachyon exception occurs
+   * @throws FileDoesNotExistException if the path does not exist
+   */
+  URIStatus getStatus(TachyonURI path, GetStatusOptions options)
+      throws FileDoesNotExistException, IOException, TachyonException;
+
+  /**
+   * Convenience method for {@link #listStatus(TachyonURI, ListStatusOptions)} with default options.
+   *
+   * @param path the path to list information about
+   * @return a list of {@link URIStatus}s containing information about the files and directories
+   *         which are children of the given path
+   * @throws IOException if a non-Tachyon exception occurs
+   * @throws FileDoesNotExistException if the given path does not exist
+   * @throws TachyonException if an unexpected Tachyon exception is thrown
+   */
+  List<URIStatus> listStatus(TachyonURI path)
+      throws FileDoesNotExistException, IOException, TachyonException;
+
+  /**
+   * If the path is a directory, returns the {@link URIStatus} of all the direct entries in it.
+   * Otherwise returns the {@link URIStatus} for the file.
```

Comment:
It's not quite the same since the return type will still be a list (with just one element). I've updated the doc.